django_opensearch_dsl/documents.py

```diff
@@ -48,6 +48,7 @@ class Document(DSLDocument):
     """Allow the definition of Opensearch' index using Django `Model`."""
 
     _prepared_fields = []
+    _order_indexing_queryset: bool = True
```

Comment:
I'd prefer an attribute in the inner class `Django` (similar to `queryset_pagination`)